pkiinit/Dockerfile

```diff
@@ -23,10 +23,16 @@ LABEL license='SPDX-License-Identifier: Apache-2.0' \
 RUN set -eux; \
     apk add --no-cache ca-certificates dumb-init
 
+RUN mkdir -p /edgex/pkiinit
+
+WORKDIR /edgex
+
 # setup / install the executable (binary or shell)
-COPY pki-init.sh pki-init.sh
-RUN chmod +x pki-init.sh
+COPY pki-init.sh pkiinit/pki-init.sh
+RUN chmod +x pkiinit/pki-init.sh
+COPY pki-init pkiinit/pki-init
 
+WORKDIR /
```

Comment:
Settings workdir seems unnecessary both times.

Reply:
will remove it and test it to see if directory structure preserves the way we want or not